Exploration Queries/DnsEvents/LeastPrevClientIP-DNSNameQueryToIP.txt

```diff
@@ -1,27 +1,23 @@
-// Name: DNS Name Lookup Query from least prevalent ClientIP to remote IP Address
-// Description: Summary of Bottom 10 Client IP and Domain Names for a given remote IPAddress from DnsEvent Lookup Query data.
-//
-// Entity: IPAddress
-// Input: IPAddress
-// Output: ClientIP
-//
-// QueryPeriod:  +-3h
-//
-// Data Source: #DnsEvents
-//
+// Id: 801bacb0-612a-4195-a84f-7939cca63b92
+// DisplayName: DNS Name Lookup Query from least prevalent ClientIP to remote IP Address
+// Description: Summary of Bottom 10 Client IP and Domain Names for a given remote IPAddress from DnsEvent Lookup Query data (set time range to +-3h when running the query)
+// InputEntityType: Ip
+// InputFields: [Address]
+// OutputEntityTypes: [Ip]
```

Comment:
do we need an OutputField as well? relying on the entity as prefix feels fragile (imagine there are several fields with that prefix..)

Reply:
query writers are free to use any field they want within the entity (of course the more the better)
however, there is no need to explicitly mention them since we know they are all fields of the given OutputEntityType(s), which do not overlap

Actually, all the fields are with the exact same prefix and this is what we want to have:
- IP_Address
- IP_Location_Latitude
- IP_Location_Longitude

--> this would result in constructing a V3 IP entity with the inner fields filled as appropriate.